pkgs/development/python-modules/miniaudio/default.nix

```diff
@@ -11,15 +11,15 @@
 
 buildPythonPackage rec {
   pname = "miniaudio";
-  version = "1.46";
+  version = "1.50";
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "irmen";
     repo = "pyminiaudio";
-    rev = "v${version}";
-    sha256 = "16llwmbbd9445rwhl4v66kf5zd7yl3a94zm9xyllq6ij7vnhg5jb";
+    rev = "refs/tags/v${version}";
```

Comment:
because of me, there was a few times where this would fail because people had a branch and a tag with the same name. And github fails the request in this case.

I added refs/tags because `fetchFromGitHub` pulls metadata from github releases, so we are guaranteed to have a tag with the same git-rev name

Reply:
all `buildPythonPackages` use the `update-python-libraries` as the update script